diff --git a/src/models/app_parameter_value.rs b/src/models/app_parameter_value.rs
index 4e13770..9f95296 100644
--- a/src/models/app_parameter_value.rs
+++ b/src/models/app_parameter_value.rs
@@ -16,7 +16,7 @@ use serde::{Deserialize, Serialize};
 pub struct AppParameterValue {
     /// Value of the parameter
     #[serde(rename = "value")]
-    pub value: std::collections::HashMap<String, serde_json::Value>,
+    pub value: serde_json::Value,
     /// Targets of the parameter
     #[serde(rename = "targets")]
     pub targets: Vec<models::AppParameterTarget>,
@@ -25,7 +25,7 @@ pub struct AppParameterValue {
 impl AppParameterValue {
     /// Application Parameter Value
     pub fn new(
-        value: std::collections::HashMap<String, serde_json::Value>,
+        value: serde_json::Value,
         targets: Vec<models::AppParameterTarget>,
     ) -> AppParameterValue {
         AppParameterValue { value, targets }
